kubetest/build.go

```diff
@@ -123,6 +123,9 @@ func (b *buildIngressGCEStrategy) Build() error {
 	// Currently, this is the only strategy.
 	var target = "push-e2e"
 
+	// Make sure we are in the ingress-gce repo before getting the image tag.
+	finishRunning(exec.Command("cd", k8s("ingress-gce")))
```

Comment:
probably add error checks for all the exec in this function

also I suspect this will change cwd and lead to other failures, I would cd back after the `git` command

Reply:
edit: you should just use `os.Chdir` here

Reply:
Done and done